app/src/components/screens/Notifications/NotificationsList.js

```diff
@@ -28,16 +47,18 @@ const NotificationsList = ({ old, recent, list, onMarkAsViewd }) => {
       return (
         <View style={styles.titleContainer}>
           <Text style={styles.title}>{section.title}</Text>
-          <TouchableOpacity
-            style={styles.markAsReadContainer}
-            onPress={() => onMarkAsViewd()}>
-            <Icon
-              type="MaterialIcons"
-              name="check-circle"
-              style={styles.markAsReadIcon}
-            />
-            <Text style={styles.markAsReadText}>Marcar como leídas</Text>
-          </TouchableOpacity>
+          {!isAllViewed ? (
```

Comment:
Es mas claro asi me parece:
```suggestion
         {isAllViewed ? null : ...
```